client/components/widget/data_viz/gviz/chart-config-directive.html

```diff
@@ -26,39 +26,112 @@
                  ng-model="ngModel.layout.columnspan">
         </div>
       </div>
-
+    </div>      
+      
+    <div class="sidebar-group">
+      <div class="sidebar-group-title">Chart</div>
+  
       <div class="sidebar-item">
         <div class="sidebar-item-label">chart type</div>
```

Comment:
These select labels are pretty visually separated from the select itself. Any chance of decreasing the margin/padding? I think it would also be useful to decorate this text in some way.